### ISSUE TITLE
Feature Request: allow scripts to be ignored from list

### ISSUE THREAD

**Issue by fiznool** [ruyadorno/ntl#8]
If you prefer to install npm modules locally, and run them via `npm run`, you can install ntl in this way:

```
npm install --save-dev ntl
```

Then you can configure in your `package.json`:

```
"scripts": {
  "start": "ntl",
  ...
}
```

This means you don't need to install `ntl` globally, and instead you can run `npm start` to bring up the ntl options.

The issue is, `ntl` will also display `start` as one of the items in the list. It would be nice to ignore this script via a command line flag. For example:

```
"scripts": {
  "start": "ntl --ignore start",
  ...
}
```

**Comment by mischah** [comment]
👍 for the blacklist option.

Excluding `npm start` isn’t possible because it’s just a personal use case.

I just wonder if this should be another key in the `package.json` or if it would make more sense to have an option for that like:

    -b, --blacklist       Exclude scripts from the list      [list]

or: 

    -e, --exclude       Exclude scripts from the list      [list]


Example: 

    ntl --exclude=precommit,ntl

**Comment by mischah** [comment]
Hej @ruyadorno,

Would you accept a PR for that? :octocat:

**Comment by ruyadorno** [comment]
sure thing! my thoughts:

- naming-wise I think either `--exclude` or `--ignore` are much better options
- ideally we should support both `package.json` field AND cli options (but if you want to start with just one that's fine 👍 )
- another improvement I think could be useful is to be able to accept `*` in order to ignore a given prefix/suffix e.g: `--ignore=pre*` to ignore all pre-scripts

**Comment by mischah** [comment]
[@paquan](https://github.com/paquan) and me are currently working on this.

We have decided to name the option `-e` --> `--exclude` to prevent an alias name clash with `-i` -> `--info`.

We skip support of defining the excludes via an `exclude` key in the `package.json` because this should be done with #23.

**Comment by mischah** [comment]
PR is created :octocat:

**Comment by ruyadorno** [comment]
Landed on `v3.2.0`

thank you so much @mischah @Paquan 🎉

**Comment by mischah** [comment]
It was our pleasure and we hat lots of fun working together on this 😄
